internal/provider/resources/deployment.go

```diff
@@ -753,24 +770,6 @@ func (r *DeploymentResource) Read(ctx context.Context, req resource.ReadRequest,
 		return
 	}
 
-	parametersByteSlice, err := json.Marshal(deployment.Parameters)
-	if err != nil {
-		resp.Diagnostics.Append(helpers.SerializeDataErrorDiagnostic("parameters", "Deployment parameters", err))
-	}
-	model.Parameters = jsontypes.NewNormalizedValue(string(parametersByteSlice))
-
-	jobVariablesByteSlice, err := json.Marshal(deployment.JobVariables)
-	if err != nil {
-		resp.Diagnostics.Append(helpers.SerializeDataErrorDiagnostic("job_variables", "Deployment job variables", err))
-	}
-	model.JobVariables = jsontypes.NewNormalizedValue(string(jobVariablesByteSlice))
-
-	parameterOpenAPISchemaByteSlice, err := json.Marshal(deployment.ParameterOpenAPISchema)
-	if err != nil {
-		resp.Diagnostics.Append(helpers.SerializeDataErrorDiagnostic("parameter_openapi_schema", "Deployment parameter OpenAPI schema", err))
-	}
-	model.ParameterOpenAPISchema = jsontypes.NewNormalizedValue(string(parameterOpenAPISchemaByteSlice))
-
```

Comment:
There were two identical sections like this, so I moved it to the `CopyDeploymentToModel` function (2a0396c).